plugin/storage/kafka/options.go

```diff
@@ -43,13 +44,17 @@ const (
 	suffixCompression      = ".compression"
 	suffixCompressionLevel = ".compression-level"
 	suffixProtocolVersion  = ".protocol-version"
+	suffixLinger           = ".linger"
```

Comment:
I would prefer we use `batch-` prefix for these new switches, to group them together in `help` output.

Reply:
Changed to --kafka.producer.batch-linger